src/components/NFTFeatured/index.vue

```diff
@@ -129,7 +129,7 @@ export default {
       event.stopPropagation();
       if (this.readOnly) return;
 
-      if (!this.hasLoggedIn) {
+      if (!this.loginAddress || this.loginAddress !== this.getAddress) {
```

Comment:
This can be a getter in store instead?

Reply:
Possible getter name: `walletHasLoggedIn`